### DISCUSSION TITLE
CreateChromatinAssay function, min.features and min.cells arguments?

### DISCUSSION THREAD

**Discussion by darya-oveisi** [stuart-lab/signac#1250] [General]
Thank you very much for your great package. 

I have a question about the min.features and min.cells arguments in the CreateChromatinAssay function.
I have a count matrix (raw count) with 43633 features and 20480 cells in which I only have 44 cells with lower than 200 counts.

```
> table(rowSums(counts)>=10)
TRUE
43633
> table(colSums(counts)>=200)
FALSE  TRUE
44 20436
```
Based on this, I thought if I create a chromatin assay using min.cell=10 and min.features=200, only 44 cells would be removed. However, when I created the object using the following code I received an object with 43611 features for 2107 cells, which means instead of 44 cells, 18373 cells have been removed. 

```
> chrom_assay <- CreateChromatinAssay(
  counts = counts,
  sep = c(":", "-"),
  genome = 'mm10',
  fragments = 'data/fragments.tsv.gz',
  min.cells = 10,
  min.features = 200)
> chrom_assay
ChromatinAssay data with 43611 features for 2107 cells
Variable features: 0
Genome: mm10
Annotation present: FALSE
Motifs present: FALSE
Fragment files: 1
```

I was wondering if there is something else happening in the CreateChromatinAssay function that I missed? Any help would be appreciated.

**Comment by timoast** [comment]
`min.features` is the minimum number of unique features detected in the cell, not the total counts for the cell. If you have peaks with >1 counts in the cell, it's likely that more cells will be removed. You should instead check `table(colSums(counts>0)>=200)`

**Reply by darya-oveisi** [reply]
Thank you for the explanation. It helped a lot.
